doc/requirements.txt

```diff
@@ -3,8 +3,8 @@ pip
 appdirs
 autograd
 autoray
-jax==0.3.17
-jaxlib==0.3.15
+jax==0.4.10
```

Comment:
Is there a reason why the doc requirements were dated so far back? :thinking:

Reply:
I think just we don't consistently update doc requirements

Reply:
This only raises an error when something is deprecated or not in the lib anymore (i.e. only detects import errors), in that case because 0.3.17 did not know about sparse.BCSR